2-ui/1-document/07-modifying-document/article.md

```diff
@@ -61,13 +61,13 @@ div.className = "alert alert-success";
 div.innerHTML = "<strong>Hi there!</strong> You've read an important message.";
 ```
 
-之后，我们就有拥有一个 DOM 元素。现在这个元素已经有一个保存类名的变量和一个保存文字信息的变量，但是在页面上依然看不到我们想要的内容，因为它还没有被插入到页面中。
+之后，我们就有拥有一个 DOM 元素。现在这个元素仅仅是一个保存类名和文字信息的变量，在页面上依然看不到我们想要的内容。因为它还没有被插入到页面中。
```

Comment:
最新原文: `After that, we have our DOM element ready. Right now it is just in a variable and we cannot see it. That is because it's not yet inserted into the page.`
[现在这个元素仅仅是一个保存类名和文字信息的变量，在页面上依然看不到我们想要的内容。]
-> [现在这个元素仅仅存于一个变量中，我们还不能在页面上看到它。]